doc/src/input/mc.md

```diff
@@ -21,32 +21,42 @@ center of mass.
 Currently, all Monte-Carlo simulations are carried out using Metropolis
 acceptance criteria.
 
-A sample input for a Monte-Carlo simulation (in the NPT ensemble) can look like
-so:
+You can add all necessary information after the `[simulations.propagator]` label.
+
+- Needed keys:
+    * `type = "MonteCarlo"`
+    * `temperature` (string): System temperature. The string contains the
+temperature with unit.
+
+- Optional keys:
+    * `update_frequency` (positive integer): After this number of steps of a move, `delta` values
+for this move are updated. Updates use statistics of a moves' acceptance ratio so it
+is recommended to choose a sufficiently high number (>100).
+
+Naturally, Monte-Carlo simulations are carried out at constant
+temperature which is set using the `temperature` key.
+
+Different from Molecular Dynamics, Monte-Carlo simulations don't carry
+information about the velocities of particles.
+As a consequence we cannot access temperature from the kinetic energy.
+
+### Example
+A sample input for a Monte-Carlo simulation (in the NPT ensemble)
+can look like so:
 
 ```toml
 [simulations.propagator]
 type = "MonteCarlo"
 temperature = "500 K"
+update_frequency = 500
+
 moves = [
-    {type = "Translate", delta = "1 A", frequency = 2},
-    {type = "Rotate", delta = "20 deg", molecule = "CO2.xyz"},
-    {type = "Resize", pressure = "10 bar", delta = "3 A^3", frequency = 0.001},
+    {type = "Translate", delta = "1 A", frequency = 2, target_acceptance = 0.5},
```

Comment:
Sure, I can change that. From my experience, setting an acceptance (or having it set to 0.5 by the program) is the default behavior in most codes I know.